src/components/form/form.scss

```
@@ -0,0 +1,195 @@
+@use 'styles/utils';
+
+.question-form {
+  @include utils.bg-defaults(cover);
+  background-image: url("assets/form-bg.jpg");
```

Все `jpg` и `png` обязательно применять с `?as=webp` для оптимизации 

```suggestion
  background-image: url("assets/form-bg.jpg?as=webp");
```

---

исправила